components/cards/GroupOverview.tsx

```diff
@@ -0,0 +1,38 @@
+import CustomText from "@/components/ui/CustomText"
+import { StyleSheet, TouchableOpacity, View } from "react-native"
+import ChevronRightIcon from "@/components/vectors/ChevronRightIcon"
+import { theme } from "@/theme"
+
+export default function GroupOverview({
+  name,
+  members,
+  id,
+}: {
+  name: string
+  members: number
+  id: string
+}) {
+  return (
+    <TouchableOpacity style={styles.container} onPress={() => {}}>
+      <View style={{ flex: 1 }}>
+        <CustomText fontWeight="semibold">{name}</CustomText>
+        <CustomText fontFamily="writer-monov" style={styles.members}>
+          {members} members
```

Comment:
Could display "member" when we have 1 person and "members" when they are more than one?